src/Native/Bootstrap/main.cpp

```diff
@@ -310,7 +315,11 @@ int main(int argc, char* argv[])
 #endif // CPPCODEGEN
 
 #ifndef CPPCODEGEN
+#ifndef CORERT_DLL
     void * osModule = PalGetModuleHandleFromPointer((void*)&__managed__Main);
```

Comment:
`#define CORERT_ENTRYPOINT  __managed__Main /  #define CORERT_ENTRYPOINT  __managed__Startup` in the ifdef above, so that we do not need to have ifdef here?